pkg/reconciler/buildrun/resources/conditions.go

```diff
@@ -33,6 +33,7 @@ const (
 	ConditionServiceAccountNotFound  string = "ServiceAccountNotFound"
 	ConditionBuildRegistrationFailed string = "BuildRegistrationFailed"
 	ConditionBuildNotFound           string = "BuildNotFound"
+	BuildRunNameInvalid              string = "BuildRunNameInvalid"
```

Comment:
Please add documentation for the new reason to this table: https://github.com/shipwright-io/build/blob/main/docs/buildrun.md#understanding-the-state-of-a-buildrun.